scripts/create_metadata

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+# OUT_DIR is where the file gets written.
+# The main purpose of this is to make testing easier.
+OUT_DIR="${OUT_DIR:-.}"
+
+# Check if DEV_TAGS is unset, if not, set it to empty.
+if [ -z "${DEV_TAGS:-}" ]; then
+    DEV_TAGS=""
+fi
+
+# SPLIT splits a string by whitespace and removes any empty matches.
+SPLIT='split("[[:space:]]+"; "gn") | map(select(length > 0))'
```

Comment:
I needed to change the split to this, otherwise all the tabs and spaces were being included in the tag name (I discovered this by writing the tests).